torch/_jit_internal.py

```diff
@@ -219,23 +220,22 @@ def unused_method(self, x):
     return fn
 
 
-def ignore(drop_on_export=False):
+def ignore(drop=False, **kwargs):
```

Comment:
The names could still use some work I think, `ignore` + `drop` isn't really clear what's going on. Maybe there should be two separate decorators for `@ignore` and `@ignore(True)` since their behavior is pretty different

Reply:
Sure, i don't love the names either. But it's the name that we've previously published; I think having too much churn probably isn't worth slightly better names.

Reply:
Since we are changing the behavior, there will be churn either way. At least adding better names will make it nicer in the long run. I would prefer to make the names as clear as possible this time so we don't have to deal with confusion forever

Reply:
Okay - what are the better names ?

Reply:
i'm going to land soon if no one has a concrete suggestion 😛

Reply:
@ignore should stay as is, and the new one should be like @unused (following C++, which…I guess)